### PR TITLE
Feature/video subtitles

### LINKED ISSUE #2665: Translation for the Secret Recovery Phrase update
## Translate the updated text

With 2.3 update: We update Seedphrase (and inconsistent variations like seed phrase, Seed Phrase, Seed phrase, seedphrase) to Secret Recovery Phrase in English. AND the new video was added to 2.3.

Translation for this update (to release in 2.4): In the all the areas of the app where this change was made, we should update the translation file(s) for all the 10 languages we support.

## Translate the video

For 2.4: Add subtitles file to the video in all the languages we support (including English, and the 10 other languages we support)

## Announcement

* Marketing to send an announcement for the change of terminology from 'Seedphrase' to 'Secret Recovery Phrase', the motivation behind it. 

* In-app "What's new" update for this announcement
** We need copy for this, and need to send this copy for translation (via Jacob)

(NOT related to #2651)

### PR DESCRIPTION
**Pull request by sethkfman**
**Description**

Subtitle work for translations

**Checklist**

* [X] There is a related GitHub issue
* [X] Tests are included if applicable
* [X] Any added code is fully documented

**Issue**

Resolves #2665

### CODE REVIEW

File: app/components/Views/MediaPlayer/index.js [hunk 1]
```diff
@@ -15,11 +15,13 @@ const styles = StyleSheet.create({
...
 
-function MediaPlayer({ uri, style, onClose }) {
+function MediaPlayer({ uri, style, onClose, textTracks, selectedTextTrack }) {
 	const [loading, setLoading] = useState(true);
 	const [error, setError] = useState(false);
 
-	const onLoad = () => setLoading(false);
+	const onLoad = e => {
+		setLoading(false);
+	};
```
Comment (wachunei) [inline app/components/Views/MediaPlayer/index.js:24]: Is there a reason for this change? 🤔

Comment (sethkfman) [reply app/components/Views/MediaPlayer/index.js:24]: Ah, no...lingering debugging. I will remove.

**Review by rickycodes — APPROVED** [review]

**Review by ibrahimtaveras00 — CHANGES_REQUESTED** [review]
In the issue, it states, "Add subtitles file to the video in all the languages we support (including English, and the 10 other languages we support)"

However, I'm only seeing the subtitles in English

In this example I have my language set to Spanish:

Screen Shot 2021-08-05 at 5 30 51 PM

**Comment by sethkfman** [comment]
@ibrahimtaveras00 Made a small tweak you should be good to test it now.

**Review by ibrahimtaveras00 — APPROVED** [review]
Fix looks good; when OS system language = MM app language all works well 👍🏽 

The only exception is we do not have the Chinese language translations.

**Pull request merged**